### ISSUE TITLE
Pre-work Checklist: Developer: Bonnie Wolfe

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/website#4873]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [ ] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [ ] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [ ] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [ ] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [ ] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [ ] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [ ] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [ ] Signup for a [Figma](https://Figma.com) account
- [ ] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [ ] (Optional) Office Hours, Thursdays 7-8pm PST
  - [ ] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [ ] Note: There are no meetings on the 1st-7th of every month.
- [ ] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [ ] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [ ] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [ ] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [ ] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [ ] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [ ] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [ ] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [ ] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [ ] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [ ] Additional notes:
   - [ ] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [ ] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [ ] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues in the [prioritized backlog](https://github.com/hackforla/website/projects/7#column-7198257) only with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [ ] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [ ] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [ ] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by HackforLABot** [comment]
## Activity Log: ExperimentsInHonesty
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8009#issuecomment-3250785483 at 09/03/2025, 01:59 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7859#issuecomment-3258769629 at 09/05/2025, 08:23 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7460#issuecomment-3263932299 at 09/07/2025, 10:42 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7460#issuecomment-3263963743 at 09/07/2025, 11:39 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7460#issuecomment-3263974253 at 09/07/2025, 12:00 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7937#issuecomment-3268449072 at 09/08/2025, 05:32 PM PDT
- ExperimentsInHonesty unassigned: https://github.com/hackforla/website/issues/7760 at 09/17/2025, 02:00 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/2688#issuecomment-3330192019 at 09/24/2025, 11:35 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-3343882118 at 09/28/2025, 09:59 AM PDT
- ExperimentsInHonesty closed as completed: https://github.com/hackforla/website/issues/7760 at 09/28/2025, 10:36 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-3369158353 at 10/05/2025, 09:14 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-3369169954 at 10/05/2025, 09:34 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-3369178978 at 10/05/2025, 09:50 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-3394932505 at 10/12/2025, 10:00 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7460#issuecomment-3394955799 at 10/12/2025, 10:11 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7937#issuecomment-3399476038 at 10/13/2025, 05:14 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-3419809098 at 10/19/2025, 10:02 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-3419813592 at 10/19/2025, 10:08 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-3419822154 at 10/19/2025, 10:21 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-3419824071 at 10/19/2025, 10:24 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7505#issuecomment-3419841838 at 10/19/2025, 10:55 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7505#issuecomment-3419864366 at 10/19/2025, 11:30 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7505#issuecomment-3419870321 at 10/19/2025, 11:40 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-3448668524 at 10/26/2025, 09:21 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-3448684798 at 10/26/2025, 09:40 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-3448691648 at 10/26/2025, 09:51 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-3448694251 at 10/26/2025, 09:56 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-3448695260 at 10/26/2025, 09:58 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7460#issuecomment-3448696452 at 10/26/2025, 10:00 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7460#issuecomment-3448711333 at 10/26/2025, 10:21 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7460#issuecomment-3478131144 at 11/02/2025, 09:32 AM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-3478177219 at 11/02/2025, 10:39 AM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-3478185709 at 11/02/2025, 10:50 AM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-3478222809 at 11/02/2025, 11:45 AM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-3478223044 at 11/02/2025, 11:46 AM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-3508536055 at 11/09/2025, 09:12 AM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-3508564554 at 11/09/2025, 09:49 AM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7666#issuecomment-3523957697 at 11/12/2025, 01:12 PM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/2481#issuecomment-3524060883 at 11/12/2025, 01:47 PM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-3539062604 at 11/16/2025, 09:53 AM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-3539079604 at 11/16/2025, 10:10 AM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-3539093455 at 11/16/2025, 10:22 AM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-3539113286 at 11/16/2025, 10:34 AM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-3539113933 at 11/16/2025, 10:34 AM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-3539121307 at 11/16/2025, 10:43 AM PST
- ExperimentsInHonesty assigned: https://github.com/hackforla/website/issues/7087 at 11/17/2025, 08:36 PM PST
- ExperimentsInHonesty issue was closed as completed: https://github.com/hackforla/website/issues/7859 at 12/02/2025, 12:48 PM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8428#issuecomment-3712717986 at 01/05/2026, 05:19 PM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8428#issuecomment-3712731276 at 01/05/2026, 05:25 PM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8428#issuecomment-3712765549 at 01/05/2026, 05:40 PM PST
- ExperimentsInHonesty assigned: https://github.com/hackforla/website/issues/8145 at 01/14/2026, 01:37 PM PST
- ExperimentsInHonesty closed as completed: https://github.com/hackforla/website/issues/8145 at 01/14/2026, 01:37 PM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8145#issuecomment-3751828730 at 01/14/2026, 01:37 PM PST
- ExperimentsInHonesty assigned: https://github.com/hackforla/website/issues/8427 at 01/14/2026, 01:37 PM PST
- ExperimentsInHonesty closed as completed: https://github.com/hackforla/website/issues/8427 at 01/14/2026, 01:37 PM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8427#issuecomment-3751830616 at 01/14/2026, 01:37 PM PST
- ExperimentsInHonesty assigned: https://github.com/hackforla/website/issues/8426 at 01/14/2026, 01:38 PM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8426#issuecomment-3751833387 at 01/14/2026, 01:38 PM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8280#issuecomment-3751837946 at 01/14/2026, 01:40 PM PST
- ExperimentsInHonesty closed as completed: https://github.com/hackforla/website/issues/8280 at 01/14/2026, 01:40 PM PST
- ExperimentsInHonesty assigned: https://github.com/hackforla/website/issues/8279 at 01/14/2026, 01:40 PM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8279#issuecomment-3751838903 at 01/14/2026, 01:40 PM PST
- ExperimentsInHonesty closed as completed: https://github.com/hackforla/website/issues/8279 at 01/14/2026, 01:40 PM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8464#issuecomment-3770605455 at 01/19/2026, 05:10 PM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7236#issuecomment-3770683223 at 01/19/2026, 05:57 PM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/6140#issuecomment-3781350295 at 01/21/2026, 01:58 PM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7780#issuecomment-3814100137 at 01/28/2026, 01:47 PM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/4262#issuecomment-3814253765 at 01/28/2026, 02:27 PM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8442#issuecomment-3838215069 at 02/02/2026, 05:10 PM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8442#issuecomment-3842696433 at 02/03/2026, 09:37 AM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/4262#issuecomment-3850007965 at 02/04/2026, 02:12 PM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/4262#issuecomment-3850026826 at 02/04/2026, 02:17 PM PST
- ExperimentsInHonesty opened: https://github.com/hackforla/website/issues/8503 at 02/04/2026, 02:28 PM PST
- ExperimentsInHonesty assigned: https://github.com/hackforla/website/issues/8503 at 02/04/2026, 02:29 PM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8503#issuecomment-3850070576 at 02/04/2026, 02:30 PM PST
- ExperimentsInHonesty closed as completed: https://github.com/hackforla/website/issues/2608 at 02/04/2026, 04:11 PM PST
- ExperimentsInHonesty closed as completed: https://github.com/hackforla/website/issues/2106 at 02/04/2026, 04:16 PM PST
- ExperimentsInHonesty assigned: https://github.com/hackforla/website/issues/5592 at 02/04/2026, 04:36 PM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-3867648099 at 02/08/2026, 09:17 AM PST
- ExperimentsInHonesty closed as completed: https://github.com/hackforla/website/issues/1734 at 02/09/2026, 05:16 PM PST
- ExperimentsInHonesty closed as completed: https://github.com/hackforla/website/issues/5653 at 02/09/2026, 05:20 PM PST
- ExperimentsInHonesty closed as completed: https://github.com/hackforla/website/issues/5653 at 02/09/2026, 05:23 PM PST
- ExperimentsInHonesty closed as completed: https://github.com/hackforla/website/issues/1734 at 02/09/2026, 05:25 PM PST
- ExperimentsInHonesty issue was closed as completed: https://github.com/hackforla/website/issues/6140 at 02/11/2026, 01:16 PM PST
- ExperimentsInHonesty assigned: https://github.com/hackforla/website/issues/8442 at 02/16/2026, 05:01 PM PST
- ExperimentsInHonesty closed as completed: https://github.com/hackforla/website/issues/8040 at 02/25/2026, 01:47 PM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8496#issuecomment-3962380304 at 02/25/2026, 02:07 PM PST
- ExperimentsInHonesty closed as completed: https://github.com/hackforla/website/issues/8040 at 02/25/2026, 04:47 PM PST
- ExperimentsInHonesty issue was closed as completed: https://github.com/hackforla/website/issues/4944 at 03/01/2026, 11:12 AM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/4944#issuecomment-3987967642 at 03/02/2026, 05:11 PM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7678#issuecomment-4000466187 at 03/04/2026, 01:39 PM PST
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8500#issuecomment-4027728861 at 03/09/2026, 05:07 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8433#issuecomment-4042047038 at 03/11/2026, 01:36 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8433#issuecomment-4042073361 at 03/11/2026, 01:40 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8433#issuecomment-4042219107 at 03/11/2026, 02:07 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8560#issuecomment-4121604423 at 03/24/2026, 02:43 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-4150539611 at 03/29/2026, 09:48 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-4150590557 at 03/29/2026, 10:05 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8487#issuecomment-4172862217 at 04/01/2026, 01:45 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8442#issuecomment-4184322683 at 04/03/2026, 10:14 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-4189125427 at 04/05/2026, 09:12 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-4189183417 at 04/05/2026, 09:52 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-4189190800 at 04/05/2026, 09:58 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-4189306745 at 04/05/2026, 11:20 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-4189307780 at 04/05/2026, 11:21 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-4189308576 at 04/05/2026, 11:22 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-4189313243 at 04/05/2026, 11:25 AM PDT
- ExperimentsInHonesty closed as completed: https://github.com/hackforla/website/issues/6426 at 04/06/2026, 04:30 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-4231889315 at 04/12/2026, 09:11 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-4231922229 at 04/12/2026, 09:31 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-4231980205 at 04/12/2026, 09:59 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-4232027309 at 04/12/2026, 10:16 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7707#issuecomment-4240329933 at 04/13/2026, 04:48 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7707#issuecomment-4240335506 at 04/13/2026, 04:49 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7707#issuecomment-4240505620 at 04/13/2026, 05:40 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8485#issuecomment-4299573695 at 04/22/2026, 01:08 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8431#issuecomment-4299751388 at 04/22/2026, 01:30 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-4416022690 at 05/10/2026, 11:29 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-4416048098 at 05/10/2026, 11:42 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-4416069471 at 05/10/2026, 11:53 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8570#issuecomment-4445160725 at 05/13/2026, 01:58 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/4262#issuecomment-4445174003 at 05/13/2026, 02:00 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/4262#issuecomment-4445192158 at 05/13/2026, 02:03 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/4262#issuecomment-4445373590 at 05/13/2026, 02:33 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/4262#issuecomment-4445397379 at 05/13/2026, 02:37 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/4262#issuecomment-4445432372 at 05/13/2026, 02:42 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8492#issuecomment-4483238905 at 05/18/2026, 04:53 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8560#issuecomment-4538255098 at 05/25/2026, 05:06 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-4587252521 at 05/31/2026, 09:08 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-4587393409 at 05/31/2026, 09:47 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-4587396956 at 05/31/2026, 09:48 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-4587481441 at 05/31/2026, 10:24 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/6956#issuecomment-4597594443 at 06/01/2026, 05:27 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8461#issuecomment-4597713589 at 06/01/2026, 05:50 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8428#issuecomment-4597749525 at 06/01/2026, 05:58 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8428#issuecomment-4597770880 at 06/01/2026, 06:02 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/3522#issuecomment-4616368594 at 06/03/2026, 01:18 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8501#issuecomment-4616385187 at 06/03/2026, 01:20 PM PDT
- ExperimentsInHonesty closed as completed: https://github.com/hackforla/website/issues/7772 at 06/03/2026, 01:38 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8376#issuecomment-4616829850 at 06/03/2026, 02:23 PM PDT
- ExperimentsInHonesty closed as completed: https://github.com/hackforla/website/issues/8376 at 06/03/2026, 02:23 PM PDT
- ExperimentsInHonesty closed as not planned: https://github.com/hackforla/website/issues/8376 at 06/03/2026, 02:23 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8420#issuecomment-4616903091 at 06/03/2026, 02:33 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8566#issuecomment-4654967417 at 06/08/2026, 05:56 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8566#issuecomment-4655731680 at 06/08/2026, 08:23 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8566#issuecomment-4656041169 at 06/08/2026, 09:16 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8691#issuecomment-4713752077 at 06/15/2026, 05:36 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7707#issuecomment-4713766780 at 06/15/2026, 05:38 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8470#issuecomment-4793243818 at 06/24/2026, 01:14 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8691#issuecomment-4793443981 at 06/24/2026, 01:42 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8691#issuecomment-4793528584 at 06/24/2026, 01:54 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/2481#issuecomment-4793596327 at 06/24/2026, 02:01 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-4826779080 at 06/28/2026, 09:59 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-4826900853 at 06/28/2026, 10:47 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7509#issuecomment-4826979255 at 06/28/2026, 11:18 AM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/7707#issuecomment-5248307581 at 08/10/2026, 07:25 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8754#issuecomment-5516346995 at 09/02/2026, 02:01 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8754#issuecomment-5516351140 at 09/02/2026, 02:02 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8754#issuecomment-5516394652 at 09/02/2026, 02:05 PM PDT
- ExperimentsInHonesty commented: https://github.com/hackforla/website/issues/8754#issuecomment-5516425262 at 09/02/2026, 02:08 PM PDT
- ExperimentsInHonesty opened: https://github.com/hackforla/website/issues/8793 at 09/19/2026, 08:56 PM PDT
